_analyzers/search-analyzers.md

```diff
@@ -67,14 +65,99 @@ PUT testindex
 ```
 {% include copy-curl.html %}
 
-## Specifying the default search analyzer for an index
+### Example: Using edge n-gram analyzer for indexing and standard analyzer for search
 
-If you want to analyze all query strings at search time with the same analyzer, you can specify the search analyzer in the `analysis.analyzer.default_search` setting. When providing the `analysis.analyzer.default_search`, you must also provide the `analysis.analyzer.default` parameter, which specifies the [index analyzer]({{site.url}}{{site.baseurl}}/analyzers/index-analyzers/) to be used at indexing time.
+The following configuration useful for implementing autocomplete-like behavior:
```

Comment:
Could you link to autocomplete? https://docs.opensearch.org/docs/latest/search-plugins/searching-data/autocomplete/

Reply:
Could you also describe this request in a bit more detail so the user sees how it mimics autocomplete?